rebasehelper/plugins/build_log_hooks/files.py

```diff
@@ -191,6 +191,17 @@ def _get_best_matching_files_section(cls, rebase_spec_file, file):
 
         return best_match_section or rebase_spec_file.get_main_files_section()
 
+    @classmethod
+    def _sanitize_path(cls, path):
+        """Changes the path to follow Fedora Packaging Guidelines."""
+        if path.startswith('%{_mandir}'):
+            manpage_start = path.rfind('/')
+            # find 2nd occurrence of '.', replace everything after it
+            # with * wildcard.
+            extension_start = path[manpage_start:].index('.', path[manpage_start:].index('.') + 1)
+            path = path[:manpage_start+extension_start] + '*'
```

Comment:
Something like this should be sufficient:

```python
dir, name = os.path.split(path)
return os.path.join(dir, name.split('.')[:2] + '*')
```

Reply:
Thanks. Would you keep the separate function for doing this? There could be more path modifications that we are missing.

Reply:
Sure, why not.